tests/utils_tests/image_tests/test_read_image.py

```diff
@@ -61,4 +65,39 @@ def test_read_image_mutable(self):
         np.testing.assert_equal(img, 0)
 
 
+@testing.parameterize(*testing.product({
+    'size': [(48, 32)],
+    'color': [True, False],
+    'suffix': ['bmp', 'jpg', 'png'],
+    'dtype': [np.float32, np.uint8, bool]}))
+class TestReadImageDifferentBackends(unittest.TestCase):
+
+    def setUp(self):
+        self.file = tempfile.NamedTemporaryFile(
+            suffix='.' + self.suffix, delete=False)
+        self.path = self.file.name
+
+        if self.color:
+            self.img = np.random.randint(
+                0, 255, size=(3,) + self.size, dtype=np.uint8)
+        else:
+            self.img = np.random.randint(
+                0, 255, size=(1,) + self.size, dtype=np.uint8)
+        write_image(self.img, self.path)
+
+    @unittest.skipUnless(_cv2_available, 'cv2 is not installed')
+    def test_read_image_different_backends_as_color(self):
+        chainer.config.cv_read_image_backend = 'cv2'
+        cv2_img = read_image(self.path, dtype=self.dtype, color=True)
+
+        chainer.config.cv_read_image_backend = 'PIL'
+        pil_img = read_image(self.path, dtype=self.dtype, color=True)
+
+        if self.suffix != 'jpg':
+            np.testing.assert_almost_equal(cv2_img, pil_img)
```

Comment:
Can we use `assert_equal` here? I hope they are same for bmp and png.

Reply:
thanks